modules/apim_analytics_dashboard/manifests/init.pp

```diff
@@ -0,0 +1,81 @@
+# ----------------------------------------------------------------------------
+#  Copyright (c) 2018 WSO2, Inc. http://www.wso2.org
+#
```

Comment:
Copy right year should be changed to 2019

Reply:
The analytics module is being reintroduced after being removed. The content is the same as what was once there earlier.